cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/JsonUtil.java

```diff
@@ -73,13 +75,34 @@
     }
 
     public static String convertToJson(Object value) {
-        if (mapper.canSerialize(value.getClass())) {
+        return convertToJsonString(value);
+    }
+
+    public static String convertToJson(Object value, boolean useIdentation) {
+        if (useIdentation) {
+            return convertToJsonUsingFeatures(value, Feature.INDENT_OUTPUT);
+        }
+
+        return convertToJsonUsingFeatures(value);
+    }
+
+    private static String convertToJsonUsingFeatures(Object value, Feature... features) {
+        mapper.enable(features);
```

Comment:
This is not tread-safe. A new mapper should be created with the indented output feature.